NextcloudTalk/BaseChatViewController.swift

```diff
@@ -170,6 +174,24 @@ import QuickLook
 
         return button
     }()
+    
+    private lazy var voiceRecordingLockButton: UIButton = {
+        let button = UIButton(frame: .init(x: 0, y: 0, width: 44, height: 44), primaryAction: UIAction { [weak self] _ in
+            // Unused atm
+        })
```

Comment:
```suggestion
        let button = UIButton(frame: .init(x: 0, y: 0, width: 44, height: 44))
```

`primaryAction` is nullable, you can just omit it, if it's not used.